waku/waku_archive_legacy/driver.nim

```diff
@@ -55,6 +56,7 @@ method getMessages*(
     hashes = newSeq[WakuMessageHash](0),
     maxPageSize = DefaultPageSize,
     ascendingOrder = true,
+    requestId = "",
```

Comment:
Why isn't that mandatory to set here if it is with V2 method?

Reply:
> Why isn't that mandatory to set here if it is with V2 method?

Yes in this case I set it a default value to prevent changing too many tests that we will remove shortly.